package.json

```diff
@@ -63,6 +63,7 @@
     "@types/node": "^20.9.2",
     "@types/react": "^18.2.37",
     "@types/react-dom": "^18.2.15",
+    "canvas-confetti": "^1.9.1",
```

Comment:
I agree that the confetti effect itself could be done by us. Since this has been on the wishlist from UX and PM if the effort is low I decided to go with `canvas-confetti`. Having the well-supported option to render in a web worker is awesome to have and the frequent releases are hinting at a good amount of edge case issues that we would need to discover and fix ourselves.

Reply:
I agree, except that the frequent releases appear to be things like being able to set additional emoji's that are used in the confetti, rather than dealing with edge cases :P